concepticondata/conceptlists/Seifart-2015-414.tsv

```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-151	151	he	1211	HE	202
+Seifart-2015-414-152	152	she	1834	SHE	203
+Seifart-2015-414-153	153	it	169	IT	204
+Seifart-2015-414-154	154	we	1212	WE	205
+Seifart-2015-414-155	155	ye	1213	YOU	206
+Seifart-2015-414-156	156	they	817	THEY	207
+Seifart-2015-414-157	157	my hand			208
+Seifart-2015-414-158	158	thy hand			209
+Seifart-2015-414-159	159	his hand			210
+Seifart-2015-414-160	160	our hands			211
+Seifart-2015-414-161	161	your hands			212
+Seifart-2015-414-162	162	their hands			213
+Seifart-2015-414-163	163	my bow			214
+Seifart-2015-414-164	164	thy bow			215
+Seifart-2015-414-165	165	his bow			216
+Seifart-2015-414-166	166	our bow			217
+Seifart-2015-414-167	167	your bow			218
+Seifart-2015-414-168	168	their bow			219
+Seifart-2015-414-169	169	big	1202	BIG	220
+Seifart-2015-414-170	170	small	1246	SMALL	221
+Seifart-2015-414-171	171	cold	1287	COLD	222
+Seifart-2015-414-172	172	hot	1286	HOT	223
+Seifart-2015-414-173	173	good	1035	GOOD	224
+Seifart-2015-414-174	174	bad	1292	BAD	225
+Seifart-2015-414-175	175	white	1335	WHITE	226
+Seifart-2015-414-176	176	black	163	BLACK	227
+Seifart-2015-414-177	177	go!			228
+Seifart-2015-414-178	178	come!			229
+Seifart-2015-414-179	179	eat!			230
+Seifart-2015-414-180	180	drink!			231
+Seifart-2015-414-181	181	sleep!			232
+Seifart-2015-414-182	182	front teeth,incisors	442	FRONT TOOTH (INCISOR)	234
+Seifart-2015-414-183	183	tip of tongue	443	TIP (OF TONGUE)	235
+Seifart-2015-414-184	184	long hair	444	LONG HAIR	236
+Seifart-2015-414-185	185	lower arm	446	LOWER ARM	240
+Seifart-2015-414-186	186	wrist	799	WRIST	241
+Seifart-2015-414-187	187	lower leg	447	LOWER LEG	242
+Seifart-2015-414-188	188	body hair	189	HAIR (BODY)	243
+Seifart-2015-414-189	189	stomach	819	STOMACH	244
+Seifart-2015-414-190	190	intestines	1991	INTESTINES	245
+Seifart-2015-414-191	191	old woman	907	OLD WOMAN	246
+Seifart-2015-414-192	192	corn field	449	CORN FIELD	249
+Seifart-2015-414-193	193	stream	947	STREAM	250
+Seifart-2015-414-194	194	pebbles	961	PEBBLES	251
+Seifart-2015-414-195	195	path	2252	PATH	253
+Seifart-2015-414-196	196	shelter	1678	HUT	255
+Seifart-2015-414-197	197	this	1214	THIS	256
+Seifart-2015-414-198	198	that	78	THAT	257
+Seifart-2015-414-199	199	who	1235	WHO	258
+Seifart-2015-414-200	200	not	1240	NOT	260
+Seifart-2015-414-201	201	all	98	ALL	261
+Seifart-2015-414-202	202	many	1198	MANY	262
+Seifart-2015-414-203	203	long	1203	LONG	263
+Seifart-2015-414-204	204	bark	1204	BARK	264
+Seifart-2015-414-205	205	flesh	2259	FLESH	265
+Seifart-2015-414-206	206	blood	946	BLOOD	266
+Seifart-2015-414-207	207	grease	323	FAT (ORGANIC SUBSTANCE)	267
+Seifart-2015-414-208	208	egg	744	EGG	268
+Seifart-2015-414-209	209	horn	1393	HORN (ANATOMY)	269
+Seifart-2015-414-210	210	feather	1201	FEATHER	270
+Seifart-2015-414-211	211	claw	72	CLAW	271
+Seifart-2015-414-212	212	belly	1251	BELLY	272
+Seifart-2015-414-213	213	liver	1224	LIVER	273
+Seifart-2015-414-214	214	drink	1401	DRINK	274
+Seifart-2015-414-215	215	eat (non meat)			275
```

Comment:
This is the most generic "eat" in the list, compared to "eat meat" and "eat fruit". If @AnnikaTjuka agrees, we could map this to EAT, but I'd like to have a second opinion on this

Reply:
I like the idea that there is a special word for eating meat and not the other way round. We have to indicate if someone eats vegetarian. I'd map it for now.